src/storage/clippy.toml

```diff
@@ -0,0 +1,26 @@
+disallowed-methods = [
```

Comment:
`clippy.toml` is expected ?

Reply:
Yes, I need to add `ignore-interior-mutability = ["risingwave_hummock_sdk::key::TableKey"]` so I just copy rules from src/streaming/clippy.toml as well.